src/test/java/com/courtcanva/ccfranchise/services/FranchiseeServiceTest.java

```diff
@@ -150,9 +172,31 @@ void shouldThrowResourceNotFoundExist() {
         SuburbListPostDto suburbListPostDto = SuburbTestHelper.createSuburbListPostDto();
 
         when(franchiseeRepository.findFranchiseeById(any()))
-                .thenReturn(Optional.empty());
+                .thenReturn(empty());
         assertThrows(ResourceNotFoundException.class, () -> franchiseeService.addDutyAreas(suburbListPostDto, 6L));
 
     }
 
+    @Test
+    void shouldThrowOrderListGetDto() {
+        List<Order> orders = OrderTestHelper.OrderList();
+        List<Order> acceptedOrders = OrderTestHelper.AcceptedOrderList();
+        OrderListPostDto orderListPostDto = OrderTestHelper.createOrderListPostDto();
+
+        when(orderRepository.findByIdIn(any())).thenReturn(orders);
+        when(orderRepository.saveAll(any())).thenReturn(acceptedOrders);
+
+        OrderListGetDto orderListGetDto = franchiseeService.acceptOrders(orderListPostDto);
+        assertEquals(OrderStatus.ACCEPTED, orderListGetDto.getOrders().get(0).getStatus());
+    }
+
+    @Test
+    void shouldThrowOrderNotFoundException() {
+        OrderListPostDto orderListPostDto = OrderTestHelper.createEmptyOrderListPostDto();
+
+        when(orderRepository.findByIdIn(any())).thenReturn(new ArrayList<>());
```

Comment:
why return ArrayList here

Reply:
Because the return type of this findById is a list. Here is the test that an exception should be thrown when no order is found, so the return value is an empty list.